src/trackExtensions.ts

```diff
@@ -0,0 +1,62 @@
+import { Extension, getExtensionHash, getLogs } from '@colony/colony-js';
+import { extensionSpecificEventsListener } from './eventListener';
+
+import networkClient from './networkClient';
+import { verbose } from './utils';
+
+const supportedExtensionIds = [getExtensionHash(Extension.VotingReputation)];
+
+export default async (): Promise<void> => {
+  verbose('Fetching already installed extensions');
+
+  supportedExtensionIds.forEach(async (extensionId) => {
```

Comment:
This is going to grow super large with time, so I'm glad this will only run once at the start.

This will be a perfect candidate for when we'll start splitting up a separate "extensions"-related block ingestor